x/evm/types/msg_test.go

```diff
@@ -169,7 +169,7 @@ func TestMsgEthereumTxSig(t *testing.T) {
 
 	// require valid signature passes validation
 	msg := NewMsgEthereumTx(0, &addr1, nil, 100000, nil, []byte("test"))
-	msg.Sign(chainID, priv1.ToECDSA())
+	_ = msg.Sign(chainID, priv1.ToECDSA())
```

Comment:
```suggestion
	err = msg.Sign(chainID, priv1.ToECDSA())
	require.NoError(t, err)
```

Reply:
This was ignored before, so my idea was to keep it ignored. If you are sure that this should be tested I can add the checks.

Reply:
yes, it should be tested